app/assets/stylesheets/components/_footer.scss

```diff
@@ -23,6 +23,18 @@ body {
     flex-direction: row;
   }
 
+  @include at-media-max('tablet') {
```

Comment:
Rather than having a pair of "at max" and "at" media queries, I'd suggest rearranging this so that our styles are mobile-first, and we only have large viewport styles as overrides. I think that'd also be a bit more consistent with how these styles are already arranged.

```diff
diff --git a/app/assets/stylesheets/components/_footer.scss b/app/assets/stylesheets/components/_footer.scss
index 61e2b41a7..b20d54a0d 100644
--- a/app/assets/stylesheets/components/_footer.scss
+++ b/app/assets/stylesheets/components/_footer.scss
@@ -16,29 +16,21 @@ body {
   align-items: center;
   justify-content: center;
   flex-direction: column;
-  font-size: 0.75rem;
+  font-size: 0.9rem;
 
   @include at-media('tablet') {
     @include u-bg('primary-darker');
     flex-direction: row;
-  }
-
-  @include at-media-max('tablet') {
-    .footer__links {
-      flex-wrap: wrap;
-      @include u-padding-x(1);
-      @include u-padding-y(0);
-    }
-    a {
-      @include u-padding-y(1);
-    }
-    font-size: 0.9rem;
+    font-size: 0.75rem;
   }
 
   a {
+    @include u-padding-y(1);
     text-decoration: none;
 
     @include at-media('tablet') {
+      @include u-padding-y(0);
+
       &,
       &:visited {
         color: color($theme-link-reverse-color);
@@ -71,6 +63,12 @@ body {
 }
 
 .footer__links {
-  @include u-padding-y(1);
+  @include u-padding-x(1);
   display: flex;
+  flex-wrap: wrap;
+
+  @include at-media('tablet') {
+    @include u-padding-y(1);
+    @include u-padding-x(0);
+  }
 }
```